conan/api/subapi/audit.py

```diff
@@ -160,3 +160,5 @@ def _load_providers(providers_path):
 
 def _save_providers(providers_path, providers):
     save(providers_path, json.dumps(providers, indent=4))
+    # Make readable & writeable only by current user
+    os.chmod(providers_path, 0o600)
```

Comment:
Just need to be sure it will not break on Windows.

Reply:
Windows ignores most bits https://docs.python.org/3/library/os.html#os.chmod